common/logisticspipes/proxy/specialinventoryhandler/StorageDrawersInventoryHandler.java

```diff
@@ -102,8 +117,11 @@ public ItemStack getMultipleItems(ItemIdentifier itemIdent, int count) {
 	public HashMap<ItemIdentifier, Integer> getItemsAndCount() {
 		HashMap<ItemIdentifier, Integer> result = new HashMap<ItemIdentifier, Integer>();
 		for(int i=0; i < _drawer.getDrawerCount(); i++) {
+			if (!_drawer.isDrawerEnabled(i))
+				continue;
+
 			IDrawer drawer = _drawer.getDrawer(i);
-			if(drawer != null && drawer.getStoredItemCopy() != null) {
+			if(drawer != null && !drawer.isEmpty()) {
 				int count = drawer.getStoredItemCount();
 				ItemIdentifier ident = ItemIdentifier.get(drawer.getStoredItemCopy());
```

Comment:
I ment ItemIdentifier.get(drawer.getStoredItemCopy()); --> ItemIdentifier.get(drawer.getStoredItemPrototype());
If the itemcount is 0 it should not be mentioned in the returned map.